api/server/volume.go

```diff
@@ -951,6 +951,7 @@ func (vd *volAPI) Routes() []*Route {
 		{verb: "POST", path: backupPath("/restore", volume.APIVersion), fn: vd.cloudBackupRestore},
 		{verb: "GET", path: backupPath("", volume.APIVersion), fn: vd.cloudBackupEnumerate},
 		{verb: "DELETE", path: backupPath("", volume.APIVersion), fn: vd.cloudBackupDelete},
+		{verb: "DELETE", path: backupPath("/all", volume.APIVersion), fn: vd.cloudBackupDeleteAll},
```

Comment:
old api should be deprecated if we don't intend on using it anymore. Again this should happen through swagger api and then removed from next release.

We should also release not this as a breaking change. (same with all the other cloudsnap api renames.

Reply:
Old API will still be used.

Changes for undocumented APIs don't need to be release noted as it is expected that they can change. Once we documents them we'll release note breaking changes (we should not be making breaking changes for documented APIs in the first place)